### PR TITLE
feat: added jahia link picker to link balloon

### PR DESCRIPTION
**Pull request by r3dm1ke**
#### Source code
- [ ] I've shared and documented any breaking change
- [ ] I've reviewed and updated the jahia-depends

#### Tests
- [ ] I've provided Unit and/or Integration Tests
- [ ] I've updated the parent issue with required manual validations

> [!TIP]
> Documentation to guide the reviews: [How to do a code review](https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2064660/How+to+do+a+code+review+-+Ref+ISSOP08.A14006)

### CODE REVIEW

File: tests/cypress/e2e/siteConfig.cy.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ describe('Rich Text CKeditor 5 - Site level configuration tests', () => {
         ckeditor5.cancel();
     });
 
-    it('Opens link picker when insert a link', function () {
+    it.only('Opens link picker when insert a link', function () {
```
Comment (gflores-jahia) [inline tests/cypress/e2e/siteConfig.cy.ts:125]: Remove `only`. Also I don't know if this is a duplicate of the test you just added, so maybe ok to remove if there's overlap/duplicate. at the very least, this should be moved to link test suite.

Comment (r3dm1ke) [reply tests/cypress/e2e/siteConfig.cy.ts]: thank you, good catch; I removed it since it is an exact duplicate of my test in `link.cy.ts`

**Review by AKarmanov — APPROVED** [review]

**Review by gflores-jahia — APPROVED** [review]
